.github/scripts/tp_hooks.py

```
@@ -0,0 +1,743 @@
+            if cost_summary:
+                print(f"  [{ticket_key}] SHADOW cost: {cost_summary}")
+            return
+
+        branch = f"test-plan/{ep_slug}"
+        dest = Path(f"enhancements/{ep_slug}/TestPlan.md")
+        dest.parent.mkdir(parents=True, exist_ok=True)
+
+        import shutil
+        shutil.copy2(output, dest)
+
+        subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)